store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -985,9 +989,9 @@ func (mvcc *MVCCLevelDB) CheckTxnStatus(primaryKey []byte, startTS uint64, curre
 			// If this is a large transaction and the lock is active, push forward the minCommitTS.
 			// lock.minCommitTS == 0 may be a secondary lock, or not a large transaction.
 			if lock.minCommitTS > 0 {
-				// We must guarantee the invariance lock.minCommitTS >= startTS + 1
-				if lock.minCommitTS < startTS+1 {
-					lock.minCommitTS = startTS + 1
+				// We must guarantee the invariance lock.minCommitTS >= callerStartTS + 1
+				if lock.minCommitTS < callerStartTS+1 {
+					lock.minCommitTS = callerStartTS + 1
```

Comment:
update to the greater currentTS can reduce write operations.

Reply:
Done